package.json

```diff
@@ -29,8 +29,14 @@
   },
   "homepage": "https://github.com/sag1v/lessc-glob",
   "dependencies": {
-    "cpx": "^1.5.0",
     "chalk": "^1.1.3",
-    "less-css-stream": "^1.0.0"
+    "cpx": "^1.5.0",
+    "less-css-stream": "^2.0.0"
+  },
+  "peerDependencies": {
+    "less": ">=1.0.0"
+  },
+  "devDependencies": {
+    "less": "^3.10.3"
```

Comment:
Why is `less` is showing up twice?

Reply:
I'll remove the devDependencies , since there are no tests